src/MarginTrading.Services/AccountManager.cs

```diff
@@ -81,23 +87,69 @@ private Task WriteAccountsStats(IReadOnlyList<MarginTradingAccount> accounts)
             return _statsRepository.InsertOrReplaceBatchAsync(stats);
         }
 
+
+
+        private Task WriteAccountsStatsReports(IReadOnlyList<MarginTradingAccount> accounts)
```

Comment:
WriteAccountsStats creates very similar objects. May be it is better to use calculated values? Also need to think about performance here....